packages/react-components/src/components/Card/Card.module.scss

```diff
@@ -9,7 +9,6 @@ $base-class: card;
   border-radius: 4px;
   background-color: var(--background);
   padding: 16px;
-  min-width: 320px;
```

Comment:
Shouldn't we have some sort of `min-width`?  How will the card render if the content is lacking?

Reply:
Ok, I've tested this solution and there seems to be no negative impact after removing `min-width`.

Reply:
👌🏼